recipe/meta.yaml

```diff
@@ -45,17 +43,19 @@ requirements:
     - {{ posix }}coreutils         # [win]
     - {{ posix }}zip               # [win]
     - cross-r-base {{ r_base }}    # [build_platform != target_platform]
-    - libgomp                      # [linux]
+    - libgomp                      # [not osx]
     - llvm-openmp                  # [osx]
   host:
+    - libgomp                      # [not osx]
+    - llvm-openmp                  # [osx]
```

Comment:
Would think changing the selector in `requirements/build` would have been sufficient here. Is there a reason it was not?